src/olympia/devhub/file_validation_annotations.py

```diff
@@ -238,7 +101,7 @@ def annotate_akismet_spam_check(results, akismet_results):
                 msg_id='akismet_is_spam_%s' % field)
 
 
-def annotate_search_plugin_validation(results, file_path, channel):
+def annotate_search_plugin_validation(results, file_path, is_listed):
```

Comment:
Can we leave it as `channel` please?  It doesn't appear to be a necessary change.

Really, I was supposed to change all the functions to use `channel` rather than `is_listed` when I implemented version channels but when faced with the huge number of validation tests that specified `is_listed` I gave up.  So if anything, code that still uses `is_listed` should be changed to use `channel` rather than the other way around.

Reply:
Gotcha, I'll do that!

Reply:
Fixed in d86e43092c (hoping that this doesn't break too much)